docs/src/api/params.md

```diff
@@ -213,6 +213,11 @@ Whether to ignore HTTPS errors during navigation. Defaults to `false`.
 
 Toggles bypassing page's Content-Security-Policy.
 
+## context-option-baseUrl
+- `baseUrl` <[string]>
```

Comment:
I now we have `waitForURL` with all caps. Do we have a `Url` precedent in our api?